README.md

```diff
@@ -1,19 +1,6 @@
-## React sc-web
+# react-sc-web
 
-- [Quick start](#quick-start)
-- [Installation](#installation)
-  * [Production](#production)
-  * [Development](#development)
-- [Develop with Docker in another project](#develop-with-docker-in-another-project)
-- [Configuration](#configuration)
-- [Recommendations](#recommendations)
-- [Files structure](#files-structure)
```

Comment:
why is it removed?

Reply:
It is extra. GitHub component for Readme provides button to show and select sections